src/extension/importer/rest/src/main/java/org/geoserver/importer/rest/ImportController.java

```diff
@@ -63,8 +68,12 @@ public ImportController(Importer importer) {
                 if (t instanceof ValidationException) {
                     throw new RestException(t.getMessage(), HttpStatus.BAD_REQUEST, t);
                 } else {
+                    LOGGER.log(
```

Comment:
Odd... are you sure it's needed? The REST machinery should be logging thrown exceptions....